test/AccessControl.js

```diff
@@ -1,111 +1,825 @@
 import { expect } from "chai";
 import EvmError from "./helpers/EVMThrow";
 import { eventValue } from "./helpers/events";
```

Comment:
rename file to AccessPolicy

Reply:
I'd keep old name. Many components are tested here, besides AccessPolicy implementation (like AccessControlled etc.)